### ISSUE TITLE
XR005 False positive on multi-line descriptions

### ISSUE THREAD

**Issue by armsnyder** [bflad/tfproviderlint#240]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### tfproviderlint and terraform-plugin-sdk Version

tfproviderlint v0.27.0
terraform-plugin-sdk v1.16.1

### Affected Check(s) or Function(s)

* XR005

### Expected Behavior

```go
func g() *schema.Resource {
	return &schema.Resource{
		Description: "Line one\n\n" +
			"Line two",
		Read:   readFunc,
		Create: createFunc,
		Schema: map[string]*schema.Schema{},
	}
}
```

Should be accepted. (It is common for resource descriptions to be long and need to be broken up.)

### Actual Behavior

The linter complains.

```
XR005: resource should configure Description
```

I will try to resolve this and may submit a PR sometime this week.
